### ISSUE TITLE
search is searching alternatively spelled tags in transcriptions, but the actual word (spelled as it should be) is not highlighted

### ISSUE THREAD

**Issue by foxm** [blakearchive/archive#224]
search 'tiger', see An Island In the Moon result

**Comment by queryluke** [comment]
@foxm this is going to be tough. The highlight is a javascript filter I wrote. It searches for the search string and wraps the string in a ```<span class="highlight"></span>```

There are 2 ways to fix this but each have a different result.

Option 1: alternative spelling should NOT be returned as a positive result.
- This would involve some stripping of tags before indexing in the solr import scripts.

Option 2: alternative spelling should be returned as a result (as it works now)
- I will need a list of all possible alternate spellings.

**Comment by foxm** [comment]
@queryluke i think we should go with option 2. the alternative spellings are in the BADs. for example:

\<desc id="songsie.l.illbk.36" dbi="songsie.L.P36-42"\>
   \<objtitle\>
      \<title type="transcribed"\>The Tyger\</title\>
      \<title type="alt"\>The Tiger\</title\>

**Comment by queryluke** [comment]
So, after digging around, I found out there are multiple issues in play. 

Concerning this particular bug, within some object's text (e.g. An Island in the Moon), there are <choice> tags. With children <orig> and <reg>/<corr> tags. I assume meaning "original" and "regular"/"correct". I've adjusted the highlight filter to account for these spelling variations. But this is really a stop gap until a better solution is found.

I've made a new issue #240 which we can assess later.
